bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,387 @@
+# Importing everything
+import asyncio
+import json
+import logging
+import random
+from calendar import month_name
+from collections import defaultdict
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import Dict, Union
+
+from async_rediscache import RedisCache
+from discord import Embed, Reaction, TextChannel, User
+from discord.colour import Colour
+from discord.ext import tasks
+from discord.ext.commands import Bot, Cog, Context, group
+
+from bot.constants import Channels, Client, Colours, Month
+from bot.utils.decorators import InMonthCheckFailure
+
+logger = logging.getLogger(__name__)
+MONTH_TO_LIMIT_TO = Month.OCTOBER
+
+EMOJIS_VAL = {  # the value of the emojis
```

Comment:
```suggestion
EMOJIS_VAL = {
```
Constant name already say that